test/parser.util.test.ts

```diff
@@ -23,11 +24,11 @@ describe("Parser", () => {
         expect(firstItem).toHaveProperty("URL", "http://www.ncbi.nlm.nih.gov/pubmed/12653513");
         expect(firstItem).toHaveProperty("volume", "43");
         expect(firstItem).toHaveProperty("issue", "2");
-        expect(firstItem).toHaveProperty("date", {
-            year: 2005,
-            month: 2,
-            day: 14
-        });
+
+        // Compare Dates
+        let date = getDate(firstItem);
```

Comment:
Here as well, the date is never modified, which is good, so you can make it const.